package/yast2.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Jan  5 11:58:05 UTC 2017 - mfilka@suse.com
+
+- bnc#956755
+  - Use full type detection when caching network configuration to
+    solve ambiguos configurations (like bridge over wlan).
```

Comment:
This is an internal module which is used mainly by other yast parts. And the change is implementation detail bcs in perfect code internal sorting of NetworkInterfaces' cache would be really internal. Sadly we more or less expose it's organization, so any change in it is public as well.